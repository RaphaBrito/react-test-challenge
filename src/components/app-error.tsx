export function AppError() {
  return <h1>Que pena, algo de errado aconteceu :(</h1>;
}
