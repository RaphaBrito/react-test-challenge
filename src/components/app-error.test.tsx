import { expect, it } from "vitest";

import { render } from "../test/utils";

import { AppError } from "./app-error";

it("should render correctly", () => {
  const { container } = render(<AppError />);
  expect(container).toMatchSnapshot();
});
