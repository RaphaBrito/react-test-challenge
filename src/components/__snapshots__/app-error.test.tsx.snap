// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render correctly 1`] = `
<div>
  <h1>
    Que pena, algo de errado aconteceu :(
  </h1>
</div>
`;
